website/package.json

```diff
@@ -9,6 +16,9 @@
     "rename-version": "docusaurus-rename-version"
   },
   "devDependencies": {
-    "docusaurus": "^1.12.0"
+    "docusaurus": "1.14.6"
```

Comment:
I tend to like pinning to a version (makes it less likely to run into weirdness across different systems) but happy to change if we used the `^` for a reason that I'm not aware of.